_toc.yml

```diff
@@ -198,13 +200,15 @@ entries:
               - file: docs/products/kafka/howto/get-topic-partition-details
               - file: docs/products/kafka/howto/schema-registry
               - file: docs/products/kafka/howto/change-retention-period
+
       - file: docs/products/kafka/reference
         title: Reference
         entries:
           - file: docs/products/kafka/reference/advanced-params
             title: Advanced parameters
           - file: docs/products/kafka/reference/kafka-metrics-prometheus
             title: Metrics available via Prometheus
+          - file: docs/products/kafka/reference/code-samples/java
```

Comment:
What should this be?
Can we remove it @anelook ?

Reply:
Thanks for noticing it!! removed